src/main/resources/liquibase/202309081416-add-span-id-event-record.xml

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202309081416-1" author="jcarvaja">
+        <comment>Add events.span_id column.</comment>
+        <addColumn tableName="events">
+            <column name="span_id" type="VARCHAR(50)"/>
```

Comment:
@kartikshahc This is a change I had to add to fix the deletion of stale events.
Note that I dislike having to expand the events table for a column that is already part of the `data` column (but in json format). However, in Postgresql, we could query directly the `data` column in json, but not in H2 (for testing). 

If there are other ways to query by json columns that work in both PostgreSQL and H2, I would be happy of changing this.

Reply:
Sure. I will review again, I'll retest this and see if I come up with anything else. Although I am fine with this method, I think having multiple uuids is causing me to think if there is another way.